### ISSUE TITLE
Errors of Dataset when runing training scripts

### ISSUE THREAD

**Issue by hejunqing** [Fanqi-Lin/OneTwoVLA#8]
I am trying to run the train_onetwovla_cocktail.sh and the dataset was downdoaded from hf.

Here is the structure of the dataset:
cot.json  
data  (directory)
meta (directory)

I set the cot path as the cocktail path and meet Errors when running the script:

<img width="2081" height="955" alt="Image" src="https://github.com/user-attachments/assets/04213073-92a8-4613-882f-811669c00893" />

<img width="1743" height="1033" alt="Image" src="https://github.com/user-attachments/assets/137ffa23-8135-456d-b80f-49921747ca0a" />

I found the UMIdataset needs refs files, should we set the config to false?

<img width="955" height="519" alt="Image" src="https://github.com/user-attachments/assets/bb8a69a2-7909-4530-adc3-969d6be3285a" />

**Comment by hejunqing** [comment]
Setting the use_reference_image=False does not work, I print the config:

config: UMITrainConfig(name='onetwovla_cocktail', project_name='openpi', exp_name='computing-norm', model=Pi0FuseConfig(action_dim=32, action_horizon=16, max_token_len=410, dtype='bfloat16', paligemma_variant='gemma_2b', action_expert_variant='gemma_300m', diffusion_loss_coeff=1.0), weight_loader=CheckpointWeightLoader(params_path='s3://openpi-assets/checkpoints/pi0_base/params'), lr_schedule=CosineDecaySchedule(warmup_steps=1000, peak_lr=2.5e-05, decay_steps=30000, decay_lr=2.5e-06), optimizer=AdamW(b1=0.9, b2=0.95, eps=1e-08, weight_decay=1e-10, clip_gradient_norm=1.0), ema_decay=0.99, freeze_filter=Nothing(), data=LeRobotUMIDataConfig(repo_id='umi/cocktail', assets=AssetsConfig(assets_dir=None, asset_id=None), base_config=UMIDataConfig(repo_id=None, asset_id=None, norm_stats=None, repack_transforms=Group(inputs=(), outputs=()), data_transforms=Group(inputs=(), outputs=()), model_transforms=Group(inputs=(), outputs=()), use_quantile_norm=False, action_sequence_keys=('actions',), prompt_from_task=True, local_files_only=True, state_down_sample_steps=[3, 15], image_down_sample_steps=[], action_down_sample_steps=3, getitem_type='necessary', use_val_dataset=True, val_ratio=0.05, create_train_val_split=True, norm_stats_dir=None, seed=42, use_reasoning=True, use_reference_image=False, is_computing_norm_stats=True, reasoning_json_path='/pfs/pfs-uaDOJM/home/hejunqing/.cache/huggingface/lerobot/umi/cocktail/cot.json', use_outdated_reasoning=True)), assets_base_dir='./assets', checkpoint_base_dir='./checkpoints', policy_dir=None, seed=42, batch_size=32, val_batch_size=12, num_workers=2, num_train_steps=30000, log_interval=100, save_interval=5000, keep_period=10000, val_interval=5000, overwrite=False, resume=False, wandb_enabled=True, policy_metadata=None, use_val_dataset=True, val_ratio=0.05, create_train_val_split=True, fsdp_devices=1, repo_id='umi/cocktail', state_down_sample_steps=[3, 15], image_down_sample_steps=[], action_down_sample_steps=3, getitem_type='necessary', use_reasoning=True, use_reference_image=False, use_outdated_reasoning=True, is_computing_norm_stats=True, reasoning_json_path='/pfs/pfs-uaDOJM/home/hejunqing/.cache/huggingface/lerobot/umi/cocktail/cot.json', prompt_from_task=True, lr_decay_till_end=True)

But face the same Error:

<img width="2695" height="1079" alt="Image" src="https://github.com/user-attachments/assets/b1014a46-597f-4ec1-8339-c15a9db6a7ca" />

**Comment by Catchip** [comment]
It looks like a network issue. Before running the training script, try to execute:
```shell
export HF_ENDPOINT="https://hf-mirror.com"
```
